packages/utils/interactive-hooks/src/Pressability/Pressability.types.ts

```diff
@@ -74,43 +46,93 @@ export type PressabilityConfig = Readonly<{
    * Called when the hover is deactivated to undo visual feedback.
    */
   onHoverOut?: (event: MouseEvent) => any;
-
-  /**
-   * Called when a long press gesture has been triggered.
-   */
-  onLongPress?: (event: PressEvent) => any;
-
-  /**
-   * Called when a press gestute has been triggered.
-   */
-  onPress?: (event: PressEvent) => any;
-
-  /**
-   * Called when the press is activated to provide visual feedback.
-   */
-  onPressIn?: (event: PressEvent) => any;
-
-  /**
-   * Called when the press location moves. (This should rarely be used.)
-   */
-  onPressMove?: (event: PressEvent) => any;
-
-  /**
-   * Called when the press is deactivated to undo visual feedback.
-   */
-  onPressOut?: (event: PressEvent) => any;
-}>;
-
-export type PressabilityEventHandlers = Readonly<{
-  onBlur: (event: BlurEvent) => void;
-  onClick: (event: PressEvent) => void;
-  onFocus: (event: FocusEvent) => void;
-  onMouseEnter?: (event: MouseEvent) => void;
-  onMouseLeave?: (event: MouseEvent) => void;
-  onResponderGrant: (event: PressEvent) => void;
-  onResponderMove: (event: PressEvent) => void;
-  onResponderRelease: (event: PressEvent) => void;
-  onResponderTerminate: (event: PressEvent) => void;
-  onResponderTerminationRequest: () => boolean;
-  onStartShouldSetResponder: () => boolean;
-}>;
+};
+
+export type PressableHoverEventProps = {
+  /**
+   * While the user API is onHoverIn the View event is onMouseEnter
+   */
+  onMouseEnter?: (event: MouseEvent) => any;
+
+  /**
+   * While the user API is onHoverOut the View event is onMouseLeave
+   */
+  onMouseLeave?: (event: MouseEvent) => any;
+};
```

Comment:
Where is this ViewEvent coming from? From RNW/win32/macOS?

Reply:
That's correct, though on 0.64 it may get an implementation on core at some point. The Pressability library has routing code for it, so the events are handled and work, this is simply the name of the events that get view implements.